wicket-core/src/test/js/dom.js

```diff
@@ -112,38 +112,6 @@ jQuery(document).ready(function() {
 		equal( el.style.display, '', "Wicket.DOM.show should set .style.display to ''." );
 	});
 
-	test("toggleClass() - single CSS class", function() {
```

Comment:
why those tests are removed ?

Reply:
Because I propose to remove that method.

Reply:
Let's deprecate it first.

Reply:
We can add @deprecated to those methods in Wicket-8.x, but I'm afraid nobody reads JSDoc anyway. Or log an error instead?

Reply:
Both `@deprecated` and a warning.

Reply:
Now I think we can deprecate it in 8.x and remove it for 9.0.0.